[submodule "kedama/Exception"]
	path = kedama/Exception
	url = https://github.com/KedamaOvO/Exception
